docs/concepts/json_schema.md

```diff
@@ -1029,6 +1076,93 @@ print(json.dumps(TypeAdapter(Person).json_schema(), indent=2))
 ```
 
 
+### Using `field_title_generator`
```

Comment:
I'm not sure why we have two sections of docs about one argument?

Reply:
Once for the model level and once for the model level.